addons/web/controllers/home.py

```diff
@@ -42,6 +42,8 @@ def web_client(self, s_action=None, **kw):
             return request.redirect(kw.get('redirect'), 303)
         if not security.check_session(request.session, request.env):
             raise http.SessionExpiredException("Session expired")
+        if not request.env['res.users'].sudo().browse(request.session.uid).has_group('base.group_user'):
+            return request.redirect('/web/login_successful', 303)
```

Comment:
shouldn't this be in `auth_signup` ?

Here you redirect to a route that's defined in a module from which `web` doesn't depend

Reply:
better indeed :)

one little small thing annoy me, `account_created` is defined in web, but only used for `auth_signup`

But I'm not sure we want to do an overwrite just for that, so keep it like this :)